### PR TITLE
408 missing contribs

### PR DESCRIPTION
**Pull request by foglabs**
CSV Reader was improperly initializing the hash for each Contribution or Instantiation, resulting in a failure to create more than one of each type of object, instead creating one object with the wrong data.

Now adds a new hash for each new object.

### CODE REVIEW

**Comment by foglabs** [comment]
also contributor field was not defined on contribution

File: app/services/aapb/batch_ingest/pbcore_xml_item_ingester.rb [hunk 1]
```diff
@@ -73,6 +73,7 @@ def raise_ingest_errors(object)
 
         def ingest_asset!
           attrs = AAPB::BatchIngest::PBCoreXMLMapper.new(pbcore_xml).asset_attributes
+
```
Comment (afred) [inline app/services/aapb/batch_ingest/pbcore_xml_item_ingester.rb]: remove?

File: app/services/aapb/batch_ingest/csv_reader.rb [hunk 2]
```diff
@@ -79,12 +79,18 @@ def csv_row_to_hash(data)
 
           else
 
+            # initialize array of contributions or instantiations etc
+            model_hash[model] ||= []
+
+              # (require('pry');binding.pry) if model == 'Contribution'
```
Comment (afred) [inline app/services/aapb/batch_ingest/csv_reader.rb]: remove?

File: app/services/aapb/batch_ingest/csv_item_ingester.rb [hunk 3]
```diff
@@ -93,7 +93,8 @@ def ingest_object_at(node, with_data, with_parent = false)
               # ingest asset's childrens
               node.children.each do |c_node|
                 with_data[c_node.object_class].each do |c_data|
-                  ingest_object_at(c_node,c_data,parent_node)
+
```
Comment (afred) [inline app/services/aapb/batch_ingest/csv_item_ingester.rb]: don't need extra line?

**Pull request merged**